app/javascript/components/recipes/index/filter-popup-content.vue

```diff
@@ -53,5 +57,11 @@ export default {
       },
     };
   },
+
+  mounted() {
+    this.filters.portions.max = this.actualfilters.portions.max;
+    console.log("actuales",this.actualfilters)
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  A space is required after ','            comma-spacing